gk/main.c

```diff
@@ -1038,8 +1038,10 @@ process_pkts_front(uint8_t port_front, uint8_t port_back,
 	if (num_arp > 0)
 		submit_arp(arp_bufs, num_arp, &gk_conf->net->front);
 
-	process_pkts_ipv4_acl(&gk_conf->net->front, lcore, &acl4);
-	process_pkts_ipv6_acl(&gk_conf->net->front, lcore, &acl6);
+	process_pkts_acl(&gk_conf->net->front, lcore, &acl4,
```

Comment:
I don't think so, because we need to know which ACL (IPv4 or IPv6) to use in each call of the function. If we just passed in the `struct gatekeeper_if`, we wouldn't know which ACL to use.